### PR TITLE
Update GCS IAM

### PR DESCRIPTION
**Pull request by apichick**
Changes in IAM for GCS module

---
**Checklist**

I applicable, I acknowledge that I have:
- [X] Read the [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md)
- [X] Ran `terraform fmt` on all modified files
- [X] Regenerated the relevant README.md files using [`tools/tfdoc.py`](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#fabric-tools)
- [X] Made sure all relevant tests pass

### CODE REVIEW

File: modules/gcs/main.tf [hunk 1]
```diff
@@ -128,13 +128,43 @@ resource "google_storage_bucket_object" "objects" {
   }
 }
 
-resource "google_storage_bucket_iam_binding" "bindings" {
+resource "google_storage_bucket_iam_binding" "authoritative" {
```
Comment (juliocc) [inline modules/gcs/main.tf:131]: Can you add a moved block for this?

**Review by juliocc — APPROVED** [review]
Lgtm

File: modules/gcs/main.tf [hunk 2]
```diff
@@ -128,13 +128,43 @@ resource "google_storage_bucket_object" "objects" {
...
   for_each = var.iam
   bucket   = google_storage_bucket.bucket.name
   role     = each.key
   members  = each.value
 }
 
+resource "google_storage_bucket_iam_binding" "bindings" {
```
Comment (ludoo) [inline modules/gcs/main.tf:138]: Miren, can you use a `iam.tf` file for IAM resources, like in all other modules?

File: modules/gcs/main.tf (hunk 1 above)
Comment (apichick) [reply modules/gcs/main.tf:131]: no, I can't because there is a new resource binding called exactly the same

File: modules/gcs/main.tf (hunk 2 above)
Comment (apichick) [reply modules/gcs/main.tf]: done

**Pull request merged**